### ISSUE TITLE
Try to use vue-virtual-scroll-list for concept lists

### ISSUE THREAD

**Issue by stefandesu** [gbv/cocoda#584]
One of the performance issues with Cocoda that is long known is with long concept lists. Having a large amount of Vue components slows down the application, especially for slower computers. I've tried to work around this issue before, but wasn't able to achieve any results. 

Recently, we have discovered [vue-virtual-scroll-list](https://github.com/tangbc/vue-virtual-scroll-list) and judging from first tests of mine, it should be suitable for our purposes. The only thing that's currently not possible is animated scrolling to a certain element, but that makes sense because the elements in between will not be loaded. I think it would be fine to not have animated scrolling as long as the performance actually improves.

**Comment by stefandesu** [comment]
It's now implemented and I tried to fix all the small details that broke because of it. Performance seems fine with the full list of Thema concepts (around 8500), see https://coli-conc.gbv.de/cocoda/test/?fromScheme=http%3A%2F%2Fbartoc.org%2Fen%2Fnode%2F1043. I also tried to use the full list of DDC concepts (>50,000). It was not very fast, but not unusable either, so definitely an improvement over before.
